install/kubernetes/cilium/values.yaml

```diff
@@ -1677,6 +1680,16 @@ prometheus:
   # ref: https://docs.cilium.io/en/stable/observability/metrics/
   metrics: ~
 
+# -- Grafana dashboards for cillium-agent
```

Comment:
This looks much better, thanks! There is one last typo in the generated files. Please re-run `make -C install/kubernetes` and `make -C Documentation helm-values.rst` again. Thanks!